packages/web/src/ErrorBoundary.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+type Props = {
+  children: React.ReactNode;
+};
+type State = any;
+
+export default class ErrorBoundary extends React.Component<Props, State> {
+  constructor(props: Props) {
```

Comment:
we can remove constructor

use

state = { error: null } 

instead

type State properly